src/blockchain_utilities/blockchain_export.cpp

```diff
@@ -72,11 +71,9 @@ int main(int argc, char* argv[])
   };
   const command_line::arg_descriptor<bool> arg_blocks_dat = {"blocksdat", "Output in blocks.dat format", blocks_dat};
 
-
-  command_line::add_arg(desc_cmd_sett, cryptonote::arg_data_dir, default_data_path.string());
-  command_line::add_arg(desc_cmd_sett, cryptonote::arg_testnet_data_dir, default_testnet_data_path.string());
-  command_line::add_arg(desc_cmd_sett, arg_output_file);
   command_line::add_arg(desc_cmd_sett, cryptonote::arg_testnet_on);
+  command_line::add_arg(desc_cmd_sett, cryptonote::arg_data_dir);
+  command_line::add_arg(desc_cmd_sett, arg_output_file);
```

Comment:
This line seems unchanged. Is git diff somehow confused, or some hidden character is present (e.g. carriage return)?